src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ObjectWriter/Dwarf/DwarfCfiOpcode.cs

```diff
@@ -11,6 +11,7 @@ internal enum CFI_OPCODE
         CFI_ADJUST_CFA_OFFSET,    // Offset is adjusted relative to the current one.
         CFI_DEF_CFA_REGISTER,     // New register is used to compute CFA
         CFI_REL_OFFSET,           // Register is saved at offset from the current CFA
-        CFI_DEF_CFA               // Take address from register and add offset to it.
+        CFI_DEF_CFA,              // Take address from register and add offset to it.
+        CFI_NEGATE_RA_STATE,      // Sign the return address in lr with pacibsp
```

Comment:
Should it be `paciasp`? I believe that it is the default used for signing return addresses.

Reply:
'll change it to paciasp/pacibsp after adding signing with sp feature.